index.html

```diff
@@ -215,23 +220,22 @@
           <p>You now can copy the dance URL, share it or get an animated GIF</p>
         </div>
         <div class="form">
-          <a id="sharetwitter" href="#" class="button" target="_blank"><img src="assets/twitter.gif" alt="share on twitter"></a>
-          <a id="sharefacebook" href="#" class="button" target="_blank"><img src="assets/facebook.gif" alt="share on facebook"></a>
-          <input id="asaturday-share-url" type="text" value="" readonly />
+          <a id="sharetwitter" href="#" class="button" target="_blank"><img src="assets/twitter.gif" alt="Share on Twitter"></a>
+          <a id="sharefacebook" href="#" class="button" target="_blank"><img src="assets/facebook.gif" alt="Share on Facebook"></a>
+          <input id="asaturday-share-url" type="text" value="" readonly>
           <div id="copy-toclipboard" data-clipboard-target="#asaturday-share-url" class="button copy">COPY URL</div>
           <a href="javascript:;" class="button" id="generategif">MAKE GIF</a>
         </div>
       </div>
     </div>
-
-  <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-    ga('create', 'UA-80530812-6', 'auto');
-    ga('send', 'pageview');
-  </script>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-35433268-69', 'auto');
```

Comment:
is this correct? we've a different analytics code now?

Reply:
oh weird, I think my `git pull --rebase` automatically changed this, without any merge conflicts. good catch!